src/math/hyp1f1.hh

```diff
@@ -550,6 +578,9 @@ namespace rascal {
         this->max_radial = max_radial;
         this->values.resize(max_radial, max_angular + 1);
```

Comment:
It's always `max_angular + 1` due to the convention of representing angular momentum in quantum mechanics: The angular momentum quantum number `l` goes from zero to some number `l_max`, which we always refer to as `max_angular`. That's `max_angular + 1` entries in total. As for why the radial index doesn't follow the same convention, well, that's just the convention that QUIP established. So instead of `max_radial` you should read `n_radial_channels`, perhaps.